src/theme/src/hooks/useInputAppearance.js

```diff
@@ -17,7 +18,9 @@ const hoverPlaceholderState = '&:hover::placeholder'
 const focusPlaceholderState = '&:focus::placeholder'
 
 const getInputStyles = (theme, appearance) => {
-  const { inputs } = theme
+  const { inputs: themeStyles } = theme
+  const defaultStyles = getDefaultStyles(theme)
```

Comment:
Can we instead pull this into the themes that rely on it (e.g. v5 and v6)? All styles should come from the theme itself, not from imported modules.